src/app-layout/index.tsx

```diff
@@ -11,7 +11,7 @@ import { AppLayoutProps } from './interfaces';
 import { Notifications } from './notifications';
 import { MobileToolbar } from './mobile-toolbar';
 import { useFocusControl } from './utils/use-focus-control';
-import useWindowWidth from './utils/use-window-width';
+import useDocumentWidth from './utils/use-document-width';
```

Comment:
`window.innerWidth` includes window scrollbars. So there is a calculation mistake by ~15px when finding available screen space